src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -171,6 +171,14 @@ JSON file, with the same label.
 Similarly the OPTIONAL `rec-<label>` key/value can be used to distinguish
 different reconstruction algorithms (for example ones using motion correction).
 
+#### The `ch` entity
+
+The OPTIONAL `ch-<index>` key/value can be used to distinguish channel-specific
+data from sequences not employing coil combination.
+The key `CoilString` MAY also be added in the JSON file, with a corresponding
+identifier for the channel within the coil.
+If `ch-<index>` is not used, data are assumed to be combined across channels.
```

Comment:
In the current proposal, they say "if skipped total is assumed", so my thinking was that having both channel-wise and combined data would look like:

```
func/
    sub-XXX_task-YYY_bold.nii.gz  # combined
    sub-XXX_task-YYY_ch-01_bold.nii.gz
    ...
    sub-XXX_task-YYY_ch-32_bold.nii.gz
```

First, do you think it makes sense to mix having the entity and not within the same run like that or should the specification allow key/value pairs with an index OR a label (specifically, "combined")?

Second, would a better way of saying that there's a default be like the following (from the run entity description):

> When there is only one scan of a given type the run key MAY be omitted.

So maybe:

> When only combined data are reconstructed the channel key MAY be omitted.

Reply:
> First, do you think it makes sense to mix having the entity and not within the same run like that or should the specification allow key/value pairs with an index OR a label (specifically, "combined")?

IMHO it is ok. 

> Second, would a better way of saying that there's a default be like the following (from the run entity description):
> 
> > When there is only one scan of a given type the run key MAY be omitted.
> 
> So maybe:
> 
> > When only combined data are reconstructed the channel key MAY be omitted.

`run` is a bit of a different beast IMHO to make direct analogy here. The suggested phrase sounds a bit off to me, since if data (in the file) is combined, there can be no `channel`.  Overall, I would not overdo it -- the purpose of all those suffixes is to provide high level information about the file and to disambiguate from one file to another.  So, may be we could flip it to

> When file contains data from a single channel, `ch-<index>` SHOULD be provided

Then, in the case when "combined" data is actually combined from a single channel recording (is that within even 0.1% of cases? ;)), it would be up to a user to decide to include or not the `ch` key.  If they have multiple channels,  the need for using `_ch` becomes obvious -- to disambiguate.

Reply:
Thank you! I've incorporated your suggestions.

Reply:
I am not sure whether it is the right place to post, but I would like to warn about confusing channel and coil. As we have just discussed with @tsalo on [mattermost](https://mattermost.brainhack.org/brainhack/channels/osr-emergent-qmri), channels refer to the two parts of the quadrature used by coils (see: http://mriquestions.com/real-v-imaginary.html) resulting in real and imaginary (or magnitude and phase) data.
Therefore, I suggest using `coil` rather than `ch`(annel) to refer to coils.

Reply:
I've switched from `ch` to `coil`. Thank you for the insight @tiborauer.

Reply:
@tiborauer do you have example of such data?  wouldn't you have then 2 `_ch-` files - one representing `I` and another `Q`?  sure thing then someone could transform them into "magnitude" and "phase" images, making it needing some `_coil` or another entity to represent "combination" of data from different channels? (so may be we need both `_ch` and `_coil`?)

Reply:
I am afraid I do not work with raw data (in MRI physics POV), but any data could be saved as such. 
I agree with you that channels refer to the two parts: real and imaginary (see the link in my previous comment), which can be transformed into magnitude and phase; which means that entity `part` (referring to magnitude and phase and also part of [BEP001](https://bep001.readthedocs.io/en/latest/04-modality-specific-files/01-magnetic-resonance-imaging-data.html#the-part-entity)) and `ch` (referring to real and imaginary) are different. Entity `coil`, however, should correspond to another level, namely the transmitter/receiver hardware usually containing multiple coil elements. See http://mriquestions.com/array-coils.html explaining Coil Elements > Segments > Channels; so you may need even more entities. :)
(N.B. You may also want to differentiate between transmitter and receiver coils.)

Reply:
As proposed in #508 (and #424 now), `part` supports both phase/magnitude and real/imaginary, so I'm not sure if that's still a concern.

I'm quite comfortable ignoring transmitter coils. I think it falls sufficiently outside the 80% BIDS is meant to cover. If someone is going to collect different runs with different transmitter coils, they can use `acq` to differentiate the files, I think.

Reply:
If you are happy to use the same entity `part` for phase/magnitude and real/imaginary, then it is fine. They are somewhat redundant anyway because one can compute one pair from the other pair. However, we have to be explicit about these - mutually exclusive - representations.

Distinguishing coils and channels is still a concern.

I agree with ignoring the transmitter/receiver coils. In most cases, the same coil(set) is used for both.